packages/devtools_app/lib/src/devtools_api.dart

```diff
@@ -0,0 +1,31 @@
+// DO NOT EDIT - edit devtools_server/lib/src/devtools_api.dart.
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// All server APIs prefix:
+const String apiPrefix = 'api/';
+
+// Flutter GA properties APIs:
+const String apiGetFlutterGAEnabled = apiPrefix + 'getFlutterGAEnabled';
+const String apiGetFlutterGAClientId = apiPrefix + 'getFlutterGAClientId';
+
+// DevTools GA properties APIs:
+const String apiResetDevTools = apiPrefix + 'resetDevTools';
```

Comment:
nit: use string interpolation instead of `+`

Reply:
Done.